bundles/org.openhab.core.config.discovery/src/test/java/org/openhab/core/config/discovery/internal/PersistentInboxTest.java

```diff
@@ -102,6 +103,8 @@ public void setup() {
     }
 
     @Test
+    @Disabled
+    // TODO: check if test should be removed
```

Comment:
Did you already check this?

Reply:
Yes. It's not needed any more, no guessing for properties and the logic for non-properties is the same.